docs/prerequisites/cmake.rst

```diff
@@ -9,3 +9,20 @@ package software.
 
 -  Please download and install the latest binary distribution of CMake
    for your platform from: https://cmake.org/download/
+
+
+.. figure:: ../images/cmake/cmake_release.PNG
```

Comment:
Just check the image. It's not rendering. @shubhamkarande13 can you confirm?

Reply:
https://moja-global-documentation--94.org.readthedocs.build/en/94/prerequisites/cmake.html

^Is this different than the intended change?

Reply:
My bad.